llvm/test/CodeGen/RISCV/VentusGPGPU/inline-asm-branch.ll

```diff
@@ -0,0 +1,39 @@
+
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py
+; RUN: llc -mtriple=riscv32 -mcpu=ventus-gpgpu -verify-machineinstrs < %s \
+; RUN:   | FileCheck -check-prefix=VENTUS %s
+
+define ventus_kernel void @getMax(ptr addrspace(1) %out, ptr addrspace(4) %in) {
```

Comment:
Why add these two testcases? Were there any changes made before? What is coomit id?

Reply:
No changes,just verify if inline-asm is avaliable in ventus.